torch/quantization/fx/prepare.py

```diff
@@ -163,7 +163,8 @@ def update_qconfig_for_qat(
     all_qat_mappings = get_combined_dict(
         get_default_qat_module_mappings(), additional_qat_module_mapping)
     object_type_dict = qconfig_dict.get("object_type", None)
-    for k, v in object_type_dict.items():
+    new_object_type_dict = object_type_dict.copy()
```

Comment:
Why is the copy needed here?

Reply:
re Raghu: this is for avoid iterating through the dict while changing it I think

@supriyar looks like `object_type_dict` can be None according to L165, wondering why that does not fail when we call None.items()

Reply:
I checked this, on line 165, if it is None then `object_type_dict` gets assigned an empty `OrderedDict()` so items returns empty.